### ISSUE TITLE
Engine can be started mid-air

### ISSUE THREAD

**Issue by gyrovague** [LevelPulse/A-4-Skyhawk#47]
To reproduce: stop engine while flying (rshift-end), wait for RPM to go to zero, then start engine again (rshift-home) and it starts up. It should only be possible to start the engine while on the ground with the huffer connected.

It should be quite easy to fix this, by assigning engine start in defaults.lua (keyboard and joystick) to something other than iCommandEnginesStart, and then catching that new command in engines.lua and only dispatching iCommandEnginesStart under the correct conditions.

**Comment by dboshardy** [comment]
Wouldn't it make sense that the engine can be restarted mid-air as the air
would spin the turbines allowing for the engine to start?

**Comment by indiadamjones** [comment]
![image_031](https://cloud.githubusercontent.com/assets/4664873/15802023/d6433084-2a74-11e6-85fa-b1c730abdee6.jpg)

**Comment by gyrovague** [comment]
Yes, @gospadin also pointed out on Discord yesterday that NATOPS specifies that the plane can self start while flying.

I guess, as the images @indiadamjones posted we basically have to limit self starting to those conditions, so this is then more of a TODO than a bug.
